### PR TITLE
chore: migration 3.0 - CartItemComponent

### LINKED ISSUE #9652: Migration for removed public members of CartItemComponent
```diff
CartItemComponent
-    view: EventEmitter<any>;
-    viewItem(): void;
```

See PR https://github.com/SAP/spartacus/pull/9430

### PR DESCRIPTION
**Pull request by Platonn**
closes #9652

### CODE REVIEW

**Comment by Platonn** [comment]
Needs merging https://github.com/SAP/spartacus/pull/9653 first

File: docs/migration/3_0.md [hunk 1]
```diff
@@ -65,6 +65,11 @@ There's no automation (schematics) planned for this change.
 
 - ConverterService constructor is now using `UnifiedInjector` instead of standard `Injector` 
 
+### CartItemComponent lost members
+`CartItemComponent` lost members: 
+- `@Output()` `view` - instead use `[cxModal]` directive to close modal on click link
```
Comment (marlass) [inline docs/migration/3_0.md]: Could we switch `click` and `link`? It sounds strange to me.

Comment (Platonn) [inline docs/migration/3_0.md]:
```suggestion
- `@Output()` `view` - instead use `[cxModal]` directive to close modal on link click
```

File: docs/migration/3_0.md [hunk 2]
```diff
@@ -65,6 +65,11 @@ There's no automation (schematics) planned for this change.
...
+- `viewItem()` - instead use `[cxModal]` directive to close modal on click link
```
Comment (Platonn) [inline docs/migration/3_0.md]:
```suggestion
- `viewItem()` - instead use `[cxModal]` directive to close modal on link click
```

File: projects/schematics/src/migrations/3_0/component-deprecations/data/cart-item.component.migration.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { CART_ITEM_COMPONENT } from '../../../../shared/constants';
+import { ComponentData } from '../../../../shared/utils/file-utils';
+
+export const CART_ITEM_COMPONENT_MIGRATION: ComponentData = {
+  selector: 'cx-cart-item',
+  componentClassName: CART_ITEM_COMPONENT,
+  removedProperties: [
+    {
+      name: 'view',
+      comment: `'view' output was removed. Instead use '[cxModal]' directive to close modal on click link.`,
```
Comment (Platonn) [inline projects/schematics/src/migrations/3_0/component-deprecations/data/cart-item.component.migration.ts]:
```suggestion
      comment: `'view' output was removed. Instead use '[cxModal]' directive to close modal on link click.`,
```

File: projects/schematics/src/migrations/3_0/component-deprecations/data/cart-item.component.migration.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    },
+    {
+      name: 'viewItem',
+      comment: `'viewItem' method was removed. Instead use '[cxModal]' directive to close modal on click link.`,
```
Comment (Platonn) [inline projects/schematics/src/migrations/3_0/component-deprecations/data/cart-item.component.migration.ts]:
```suggestion
      comment: `'viewItem' method was removed. Instead use '[cxModal]' directive to close modal on link click.`,
```

File: docs/migration/3_0.md (hunk 1 above)
Comment (Platonn) [reply docs/migration/3_0.md]: Done

**Review by marlass — APPROVED** [review]

**Pull request merged**